Gemfile

```diff
@@ -15,7 +15,8 @@ gem 'nokogiri', '>= 1.6.7.1'
 gem "order_management", path: "./engines/order_management"
 gem 'web', path: './engines/web'
 
-gem 'pg'
+gem 'activerecord-postgresql-adapter'
```

Comment:
This looks a bit odd to me. We have another Rails 4.2 and Ruby 2.3.0 app and we don't need this gem. Was it due to the upgrade of `pg`?

Checking the code of the gem at https://github.com/larskanis/activerecord-postgresql-adapter I don't understand it either.

Reply:
I think bundler failed when I ran it, and the output suggested we needed this.

Reply:
I think it adds compatibility for Rails 3 activerecord gems. Some of them try to include older versions of pg directly, and break.

Reply:
You can see this if you try to run the app without the adapter gem.